packages/rustc/config.toml

```diff
@@ -27,7 +27,7 @@ codegen-tests = false
 [target.x86_64-unknown-linux-gnu]
 cc = "gcc"
 cxx = "g++"
-llvm-config = "/usr/bin/llvm-config"
+llvm-config = "/usr/bin/llvm-config-3.9"
```

Comment:
@joakim-noah It was done because we had Ubuntu 16.04 in docker initially. I'll remove it, thanks!